packages/webapp/src/components/Profile/EditUser/index.jsx

```diff
@@ -44,6 +44,8 @@ export default function PureEditUser({
   const adminRoles = [1, 2, 5];
 
   const genderOptions = useGenderOptions();
+  const getGenderOptionLabel = (option) => t(option.label);
+
```

Comment:
Not in scope for this PR, but in this file there's a `getDefaultGender` function which could be simplified now to simply

```js
const getDefaultGender = () => genderOptions.find(({ value }) => value === userFarm.gender)
```

similarly to what you did in the first component for default values